File: CodeForces/201-300/228/A/228A.ts
// Problem 228A: Is your horseshoe on the other hoof?
// https://codeforces.com/contest/228/problem/A
// Compiled with: npx 9.2.0, tsc Version 5.9.3
// Executed with: node v18.20.4
// Linux version: Linux kernel 6.1.0-10-amd64
// Written on: October 23rd, 2025
// Codeforces language used: Cannot be submitted

import * as readline from 'node:readline/promises';
import { stdin as input, stdout as output } from 'node:process';

async function main() {
    const rl = readline.createInterface({ input, output });

    const line: string = await rl.question('');
    const arr: number[] = line
        .trim()
        .split(' ')
        .map(s => parseInt(s, 10));
    rl.close();

    let shoeSet = new Set<number>(arr);
    console.log(4-shoeSet.size);
}

main();

File: CodeForces/201-300/228/A/228A.py
# Problem 228A: Is your horseshoe on the other hoof?
# https://codeforces.com/contest/228/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 27th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

colors = set(map(int, input().split()))
print(4-len(colors))
